File: pingyougou_seckill_web/src/main/webapp/js/service/seckillService.js
//服务层
app.service('seckillService',function($http){
	    	
	//查询秒杀商品列表
	this.findSeckillGoodsList=function(){
		return $http.get('seckill/findSeckillGoodsList.do');
	}

    //查询秒杀商品详情
    this.findBySeckillGoodsId=function(seckillGoodsId){
        return $http.get('seckill/findBySeckillGoodsId.do?seckillGoodsId='+seckillGoodsId);
    }

    //秒杀下单
	this.saveSeckillOrder=function (seckillGoodsId) {
        return $http.get('seckill/saveSeckillOrder.do?seckillGoodsId='+seckillGoodsId);
    }
});

File: pingyougou_seckill_interface/src/main/java/com/pingyougou/seckill/service/SeckillService.java
package com.pingyougou.seckill.service;

import com.pingyougou.pojo.TbSeckillGoods;

import java.util.List;

public interface SeckillService {

    /**
     * 查询秒杀商品列表
     */
    public List<TbSeckillGoods> findSeckillGoodsFromRedis();

    /**
     * 查询秒杀商品详情
     */
    public TbSeckillGoods findBySeckillGoodsId(Long seckillGoodsId);

    /**
     * 秒杀下单
     */
    public void saveSeckillOrder(Long seckillGoodsId,String userId);
}
